docs/team/member-guide.md

```diff
@@ -0,0 +1,101 @@
+# JupyterHub team guide
+      
+This page holds resources for members of the JupyterHub and Binder teams.
+They're meant to guide team members to be happy, productive members of the
+team!
+
+## What are the team resources?
+   
+There are a few resources that are particularly useful for team members. Here's
+a quick list to get you started.
+   
+* [**The JupyterHub Team Compass**](https://jupyterhub-team-compass.readthedocs.io/en/latest/contributing.html)
+  is a repository with lots of information about team-related things. It has
+  development tips, information about team meetings, milestones and roadmaps,
+  etc.
+* [**The JupyterHub Team Compass issues**](https://github.com/jupyterhub/team-compass/issues)
+  are where we often discuss specific, actionable things related to the *team*
+  (e.g., discussing whether to change something in the team-compass repo).
+* **The JupyterHub gitter channels** are used to have synchronous conversation
+  for several projects. If a conversation will likely span multiple hours,
+  or is relevant to many people, consider opening a thread in Discourse or
+  the `team-compass` repository instead. Here are the relevant Gitters:
+
+    * [The JupyterHub Gitter](https://gitter.im/jupyterhub/jupyterhub) is for
+      discussing JupyterHub, Z2JH, TLJH, etc.
+    * [The Binder Gitter](https://gitter.im/jupyterhub/binder) is for
+      discussion with Binder users about how to use Binder.
+    * [The mybinder.org-deploy gitter](https://gitter.im/jupyterhub/mybinder.org-deploy)
+      is for discussion around **operating myinder.org**.
+
+* [**The mybinder SRE guide**](https://mybinder-sre.readthedocs.io/en/latest/)
+  is a set of resources for running mybinder.org. It contains tips, snippets,
+  lessons-learned etc for running a large, public kubernetes cluster.
+* [**The JupyterHub Team Google Drive**](https://drive.google.com/drive/u/1/folders/0B8VZ4vaOYWZ3X29KTzZSemlNSG8)
+  is a place to store documents, presentations, images, etc that are useful to
+  the team. You can put whatever you'd like here, just try to keep it organized :-)
+
+## How can I help?
+   
+As a member of the team, you are encouraged to continue
+helping in the same ways that you already have. Your contributions to
+documentation, code, etc are always welcome, along with anything in the
+["how can I contribute"](https://jupyterhub-team-compass.readthedocs.io/en/latest/contributing.html)
+guide in the team-compass.
+   
+Don't forget that, as a member of the team, you're representing the community
+when you interact with people online. Try to keep a friendly, positive
```

Comment:
```suggestion
when you interact with people (online and offline). Try to keep a friendly, positive
```

Because even IRL people deserve to be treated well :D